plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/StandardDeployJob.java

```diff
@@ -86,7 +85,8 @@ private CloudSdk getCloudSdk() {
     MessageConsole messageConsole =
         MessageConsoleUtilities.getMessageConsole(CONSOLE_NAME, null, true /* show */);
     final MessageConsoleStream outputStream = messageConsole.newMessageStream();
-    CloudSdk cloudSdk = new CloudSdkProvider().createBuilder()
+    CloudSdk cloudSdk =
```

Comment:
What would be the use case for such a headless execution? Currently users deploy by clicking on a menu item.
I'm not against removing UI dependency from the job, just curious.